sort.php

```diff
@@ -37,10 +47,18 @@ function bubbleSort($sort_array)
     return $sort_array;
 }
 
-if($argc > 1 && substr($argv[1], 0, 1) !== '-') {
-    $fileName = $argv[1];
+$optind = null;
+$options = getopt("n", [], $optind);
+if ($options) {
```

Comment:
Please explain this condition.

Reply:
whether $ptions is === true or false

Reply:
This statement is the same as when you do like this ``if ($options === true)?```

Reply:
No. getopt() doesn't return true. Please consult its manual and re-think your condition.